src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -79,7 +79,12 @@ class ClusterObserver : IDisposable
 
         public async Task Run(CancellationToken token)
         {
-            var prevStats = new Dictionary<string, ClusterNodeStatusReport>();
+            // we give some time to populate the stats.
+            await TimeoutManager.WaitFor(SupervisorSamplePeriod, token);
+            var prevStats = _maintenance.GetStats();
```

Comment:
_can_ this fail?

Reply:
This was initially my concern. But then I realized that this simply can't fail. :)